cura_app.py

```diff
@@ -207,5 +207,11 @@ def exceptHook(hook_type, value, traceback):
     import trimesh.exchange.load
     os.environ["LD_LIBRARY_PATH"] = old_env
 
+# WORKAROUND: CURA-5488
```

Comment:
```suggestion
# WORKAROUND: Cura#5488 
```
CURA-number is used for our Jira tickets (which is an internal tracker). Github uses [reponame]#[issue_nr]

Reply:
Applied without extra whitespace